besu/src/test/java/org/hyperledger/besu/cli/CommandTestAbstract.java

```diff
@@ -167,6 +168,8 @@ public abstract class CommandTestAbstract {
   @Mock
   protected Logger mockLogger;
 
+  @Mock protected BesuComponent mockContext;
```

Comment:
mix of 'Context' and 'Component' being used in the var names

Reply:
This was intentional per the description of Component above, but I am happy to change if this is misleading.  Eventually we will introduce subcomponents, which I would name something like P2PComponent, but then refer to the top-level component as a single application context.

Reply:
I guess I'd favour whatever is idiomatic for Dagger. My objection here is about having two variable names for the same thing, would prefer if we stick with one or the other everywhere.

Reply:
I'd be fine with any of mockContext, mockBesuContext, mockApplicationContext, mockBesuComponent, as long as we use the same in the prod code.

Reply:
mockBesuComponent makes sense to me